lectures/markov_chains_I.md

```diff
@@ -632,7 +646,7 @@ The distributions we have been studying can be viewed either
 1. as probabilities or
 1. as cross-sectional frequencies that the law of large numbers leads us to anticipate for large samples.
 
-To illustrate, recall our model of employment/unemployment dynamics for a given worker {ref}`discussed above <mc_eg1>`.
+To illustrate, recall our model of employment/unemployment dynamics for a given worker discussed in {ref}`Example 2: Unemployment <mc_eg1>`.
```

Comment:
@Jiarui-ZH can you update these links to use

https://sphinx-proof.readthedocs.io/en/latest/syntax.html#referencing-examples